public_html/wikiWhoProxy.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+header( 'Content-Type: application/json' );
+header( 'Access-Control-Allow-Origin: *' );
```

Comment:
Or maybe we should just restrict to Wikipedias here, rather than checking the referrer? Some conditions such as altered security settings may bypass this, while the referrer would have to be spoofed (which is also pretty easy to do...)

Reply:
It doesn't seem very likely that anyone will abuse this, so I think how you've got it is fine. That said, we'll probably not even know if someone's hitting this too much unless we add some log-checking or something.